Mathlib/Topology/Bornology/Absorbs.lean

```diff
@@ -0,0 +1,247 @@
+/-
+Copyright (c) 2023 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
```

Comment:
```suggestion
/-
Copyright (c) 2020 Jean Lo, Yury Kudryashov. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Jean Lo,Yury Kudryashov
-/
```
seems more accurate. See https://mathlib-changelog.org/v3/commit/05822370

Reply:
Indeed, I added the header as a placeholder, then forgot to update it.